tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+- name: Set login shell for user www-data
+  user: name=www-data shell=/bin/bash
+
+- name: Clone the opencraft repository
+  git: repo=https://github.com/open-craft/opencraft.git dest="{{ opencraft_root_dir }}" update=no
+  become_user: www-data
+
+- name: Install system dependencies
+  command: make install_system_dependencies chdir="{{ opencraft_root_dir }}"
+
+- name: Install virtualenv
+  pip: name=virtualenv executable=pip3
```

Comment:
Some parts here and in https://github.com/open-craft/opencraft/blob/master/bin/bootstrap are duplicate logic. This wasn't explicitly part of this PR's scope, but that's still technical debt; worth putting in a task in the backlog if you don't have time to handle it here?